routing/localchans/manager.go

```diff
@@ -106,6 +111,36 @@ func (r *Manager) UpdatePolicy(newSchema routing.ChannelPolicy,
 		return err
 	}
 
+	// Return an error if any of the channels in chansToUpdate
+	// were not found.
+	if len(chansToUpdate) > 0 {
+		var errStr string
+		for _, chanPoint := range chanPoints {
```

Comment:
`for chanPoint := range chansToUpdate {...}`

- Then you dont need to do the length check above
- also removes some unnecessary queries

Reply:
@ellemouton You're right. Should have seen this myself. Thx.